LICENSE

```diff
@@ -1,4 +1,4 @@
-// Copyright © 2010-2018 The CefSharp Authors. All rights reserved.
+// Copyright © 2017 The CefSharp Authors. All rights reserved.
```

Comment:
Should this be 2019?

Reply:
c12b2392184a49309c4bfcb8b46db8cf8c07e9fb

I was a little bit uncertain on this one so I set it to the last time it was actually modified.
However, it seems like this year can be removed as well

Reply:
The year has now been removed